hnn_core/gui/gui.py

```diff
@@ -1516,6 +1741,160 @@ def run_button_clicked(widget_simulation_name, log_out, drive_widgets,
                                           plot_type, {}, "plot")
 
 
+def _update_cell_params_vbox(cell_type_out, cell_parameters_list,
+                             cell_type, cell_layer):
+    cell_parameters_key = f"{cell_type}_{cell_layer}"
+    if "Biophysics" in cell_layer:
+        cell_parameters_key += f" {cell_type.split(' ')[0]}"
+
+    if cell_parameters_key in cell_parameters_list:
+        cell_type_out.clear_output()
+        with cell_type_out:
+            display(cell_parameters_list[cell_parameters_key])
+
+
+def _update_geometry_cell_params(net, cell_param_key, param_list):
+    cell_params = param_list
+    cell_type = f'{cell_param_key.split("_")[0]}_pyramidal'
+
+    sections = net.cell_types[cell_type].sections
+    # Soma
+    sections['soma']._L = cell_params[0].value
+    sections['soma']._diam = cell_params[1].value
+    sections['soma']._cm = cell_params[2].value
+    sections['soma']._Ra = cell_params[3].value
+
+    # Dendrite common parameters
+    dendrite_cm = cell_params[4].value
+    dendrite_Ra = cell_params[5].value
+
+    dendrite_sections = [
+        'apical_trunk', 'apical_1', 'apical_tuft', 'apical_oblique',
+        'basal_1', 'basal_2', 'basal_3'
+    ]
```

Comment:
```suggestion
    dendrite_sections = [name for name in sections.keys() if name != 'soma']
```
This list is actually missing `apical_2` so perhaps it's better to just loop over the section names

Reply:
Done